src/ucs/config/parser.c

```diff
@@ -70,6 +70,19 @@ const char *ucs_async_mode_names[] = {
 
 UCS_CONFIG_DEFINE_ARRAY(string, sizeof(char*), UCS_CONFIG_TYPE_STRING);
 
+
+typedef struct ucs_config_parse_section {
+    char name[64];
```

Comment:
where is this section name used?

Reply:
In the first condition in `ucs_config_parse_skip_section`.